akka-stream/src/main/scala/akka/stream/impl/Unfold.scala

```diff
@@ -113,20 +114,21 @@ import scala.util.{ Failure, Success, Try }
       }
 
       def onPull(): Unit = {
-        val future = f.apply(state).toCompletableFuture
```

Comment:
`.toCompletableFuture` which may throw exception, now fixed.

Reply:
Do you refer to this?

> A CompletionStage implementation that does not choose to interoperate with others may throw UnsupportedOperationException.

Can you give a concrete example of when this will happen?

Reply:
I have never encounter this as I only use Java's CompletableFuture, but seems this can happen from the javadoc.

Reply:
I don't think we should add specific handling of this until someone reports a problem

Reply:
I see, let's reopen this if an issue is reported.
I opened this because @lrytz mentioned it once.